dist/typeahead.bundle.js

```diff
@@ -1726,7 +1726,6 @@
                     if (!canceled && rendered < that.limit) {
                         that.cancel = $.noop;
                         if (!that.updateOnAsync) {
-                            that._append(query, suggestions.slice(0, idx));
```

Comment:
on 11.1 I've made the same kind of fix :  that._append(query, suggestions.slice(0, that.limit));
because on some situation the menu is not filled correctly with the remote response.